data/transactions/logic/fields.go

```diff
@@ -35,7 +35,7 @@ const (
 	Fee
 	// FirstValid Transaction.FirstValid
 	FirstValid
-	// FirstValidTime panic
+	// FirstValidTime Transaction.FirstValidTime
```

Comment:
This doesn't really serve to document the constant, because FirstValidTime isn't really a field of Transaction.

Reply:
thanks, fixed